internal/install/static_kibana_healthcheck_sh.go

```diff
@@ -0,0 +1,11 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package install
+
+const kibanaHealthcheckSh = `#!/bin/bash
+
+curl -s -f http://127.0.0.1:5601/login | grep kbn-injected-metadata 2>&1 >/dev/null
```

Comment:
BTW, do we need this?

Reply:
Yes, this way we're checking when Kibana is ready. Normally the port is open and the HTTP server is exposed, but it responds with something like "not ready yet".

Reply:
I see, thanks.

Reply:
In this case, should we add a `set -e` at the top of this bash script?

Reply:
I forgot about this! Thanks